src/fix_langevin.cpp

```diff
@@ -211,6 +223,21 @@ int FixLangevin::setmask()
 
 void FixLangevin::init()
 {
+  if (gjfflag){
+    if (t_period*2 == update->dt)
+      error->all(FLERR,"Fix langevin gjf cannot have t_period equal to dt/2");
+
+    // warn if any integrate fix comes after this one
+    int before = 1;
+    int flag = 0;
+    for (int i = 0; i < modify->nfix; i++) {
+      if (strcmp(id,modify->fix[i]->id) == 0) before = 0;
+      else if ((modify->fmask[i] && strcmp(modify->fix[i]->style,"nve")==0) && before) flag = 1;
```

Comment:
this `strcmp()` test is not compatible with using various styles with suffixes. better would be to use: something like `utils::strmatch(modify->fix[i]->style,"^nve")`